metricbeat/module/mongodb/status/_meta/data.json

```diff
@@ -1,209 +1,207 @@
+                "collection": {
+                    "acquire": {
+                        "count": {
+                            "r": 232,
+                            "w": 2
+                        }
+                    },
+                    "deadlock": {},
+                    "wait": {}
+                },
+                "database": {
+                    "acquire": {
+                        "count": {
+                            "R": 1,
+                            "W": 7,
+                            "r": 232
+                        }
+                    },
+                    "deadlock": {},
+                    "wait": {}
+                },
+                "global": {
+                    "acquire": {
+                        "count": {
+                            "W": 3,
+                            "r": 955,
+                            "w": 7
+                        }
+                    },
+                    "deadlock": {},
+                    "wait": {}
+                },
+                "meta_data": {
+                    "acquire": {
+                        "count": {
+                            "w": 1
+                        }
+                    },
+                    "deadlock": {},
+                    "wait": {}
+                }
+            },
+            "memory": {
+                "bits": 64,
+                "mapped": {
+                    "mb": 0
+                },
+                "mapped_with_journal": {
+                    "mb": 0
+                },
+                "resident": {
+                    "mb": 59
+                },
+                "virtual": {
+                    "mb": 930
+                }
+            },
+            "network": {
+                "in": {
+                    "bytes": 3712
+                },
+                "out": {
+                    "bytes": 358612
+                },
+                "requests": 127
+            },
+            "ops": {
+                "counters": {
+                    "command": 64,
+                    "delete": 0,
+                    "getmore": 0,
+                    "insert": 0,
+                    "query": 1,
+                    "update": 0
+                },
+                "latencies": {
+                    "commands": {
+                        "count": 63,
+                        "latency": 7480
+                    },
+                    "reads": {
+                        "count": 0,
+                        "latency": 0
+                    },
+                    "writes": {
+                        "count": 0,
+                        "latency": 0
+                    }
+                },
+                "replicated": {
+                    "command": 0,
+                    "delete": 0,
+                    "getmore": 0,
+                    "insert": 0,
+                    "query": 0,
+                    "update": 0
+                }
+            },
+            "process": "mongod",
```

Comment:
To me it sounds superfluous to keep having a custom field here, when we have `process.name` defined in ECS. There's a few places where `process.name` is set, even if there's nothing else being populated in the field set, in the Filebeat modules (system, stack, haproxy).

I don't feel super strongly about it, though.

Reply:
Changed it to process.name.